README.md

```diff
@@ -511,6 +514,55 @@ name:      lcd_1602_i2c
 web site:  https://github.com/raspberrypi/pico-examples/tree/HEAD/i2c/lcd_1602_i2c
 ```
 
+## verify
+
+This command allows you to verify the contents of Flash/SRAM on the device matches a file, similar to the `--verify` option for `save` and `load`.
```

Comment:
"allows you to verify the contents" -> "allows you to verify that the contents" ?
(and yes, I realise that fixing this will mean editing `main.cpp` rather than directly editing `README.md`)